Week1/typeCheck.js

```diff
@@ -0,0 +1,44 @@
+let myBoolean = false;
+console.log("The value of my variable myBoolean is: " + myBoolean);
+console.log("The type of the variable is boolean");
+console.log(typeof myBoolean);
+let myText = "hello";
+console.log("The value of my variable myText is: " + myText);
+console.log("The type of the variable is string");
+console.log(typeof myText);
+let myNumber = 24;
+console.log("The value of my variable myNumber is: " + myNumber);
+console.log("The type of the variable is number");
+console.log(typeof myNumber);
+let myArray = ['apple','orange'];
+console.log("The value of my variable myArray is: " + myArray);
+console.log("The type of the variable is object");
+console.log(typeof myArray);
+if (typeof myArray == typeof myBoolean) {
+    console.log('SAME TYPE');
+}
+else {
+    console.log('DIFFERENT TYPE')
+};
+
+if (typeof myText == typeof myArray) {
+    console.log('SAME TYPE');
+}
+else {
+    console.log('DIFFERENT TYPE')
+};
+
+if (typeof myNumber == typeof myText) {
+    console.log('SAME TYPE');
+}
+else {
+    console.log('DIFFERENT TYPE')
+};
+
+if (typeof myNumber == typeof myBoolean) {
+    console.log('SAME TYPE');
+}
+else {
+    console.log('DIFFERENT TYPE')
+};
```

Comment:
I think you forgot to compare some types, such as `myArray` with `myNumber`

Reply:
Hello Marciano;

First of all thank you very much for your comments and feedbacks. I really appreciate it! Homework commands to create different types of variables so It was already obvious that all the result will be "DIFFERENT TYPE". so I thought it would be enough to test just some of them. :)

Thenak you very much again! Have a nice week

Reply:
Thanks @lizozo for implementing the changes suggested by @marcianoviereck Also great to see that you are responding to his feedback - very professional!